apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/dao/state/phasing/PhasingPollVoterTable.java

```diff
@@ -67,7 +66,7 @@ public void save(Connection con, PhasingPollVoter voter) throws SQLException {
         }
     }
 
-    public DbIterator<Transaction> getVoterPhasedTransactions(long voterId, int from, int to) throws SQLException {
+    public List<Transaction> getVoterPhasedTransactions(long voterId, int from, int to, int height) throws SQLException {
```

Comment:
Connection con = null;
can be moved into try-with-resources code.

Reply:
Fixed